index.html

```diff
@@ -0,0 +1,85 @@
+<?php
```

Comment:
Make sure the file extension of this file is .php rather than .html, otherwise your php wont work.